tests/test_bqeth_subscription.py

```diff
@@ -35,7 +36,16 @@
 YELLOW_PERIOD = ONE_DAY
```

Comment:
Just want to check that the base fee rate will increase (e.g. on a yearly basis) regardless of whether the ritual ends up falling into the yellow or red grace periods, and regardless of when they renew.

Reply:
Yes, I think it's the case because base fee rate increase depends exclusively on which period number we are at the moment, and that only depends on the period duration and the initial subscription timestamp. In any case, let's wait for confirmation from Vicky

Reply:
@cygnusv 100% correct